tools/prepublish.ts

```diff
@@ -37,44 +37,36 @@ const extract = (input, opts, callback) => {
       });
 };
 
-async.series(
-    [
-      (next) => {
-        require('child_process').exec('rm -rf google', next);
-      },
+async function main() {
+  await require('child_process').exec('rm -rf google');
```

Comment:
Pretty sure this doesn't return a promise.  You will need to use `util.promisify` to wrap any of these